segmentstore/server/src/test/java/io/pravega/segmentstore/server/SegmentStoreMetricsTests.java

```diff
@@ -230,8 +230,8 @@ public void testOperationProcessorMetrics() throws Exception {
                 new TestCompletableOperation(30));
         op.operationsFailed(opf);
         assertEquals(20, (int) MetricRegistryUtils.getTimer(MetricsNames.OPERATION_LATENCY, containerTag).totalTime(TimeUnit.MILLISECONDS));
-        SegmentStoreMetrics.reportOperationLogSize(1000, 1);
-        AssertExtensions.assertEventuallyEquals(true, () -> MetricRegistryUtils.getGauge(MetricsNames.OPERATION_LOG_SIZE).value() == 1000, 2000);
+        SegmentStoreMetrics.reportOperationLogSize(1000, containerId);
+        AssertExtensions.assertEventuallyEquals(true, () -> MetricRegistryUtils.getGauge(MetricsNames.OPERATION_LOG_SIZE, containerTag(containerId)).value() == 1000, 2000);
```

Comment:
Why this only fails sometimes? If the container is wrong, shouldn't it be failing consistently?

Reply:
If this test has been passing at times, it would mean, when the gauge was retrieved only the container for which the gauge was created was registered. Might have to do with timing of the test. But with the change i see the test consistently passes on internal Jenkins builds and GH Actions.